src/components/tx/modals/NftTransferModal/SendNftForm.tsx

```diff
@@ -177,7 +170,7 @@ const SendNftForm = ({ params, onSubmit }: SendNftFormProps) => {
                     <MenuItem key={item.address + item.id} value={item.id}>
                       <NftMenuItem
                         image={item.imageUri || item.logoUri}
-                        name={item.name || item.tokenName}
+                        name={item.name || `${item.tokenName || item.tokenSymbol || ''} #${item.id}`}
```

Comment:
This can potentially leave an empty space, but it should hopefully not happen in practice.